src/scss/theme.scss

```diff
@@ -392,11 +402,18 @@ div.prev-next-bottom {
 /* Header Styling */
 
 h1 {
-  font-size: 3rem;
+  font-size: 2.5rem;
 }
 
 h2 {
-  font-size: 2.5rem;
+  font-size: 1.75rem;
+  margin-top: 0.3em;
```

Comment:
Similarly, I thought we discussed doing this for all headers below h1, not just h2.